### DISCUSSION TITLE
Issue with Student Benefits Activation after Verification

### DISCUSSION THREAD

**Discussion by Bend1031** [community/community#65408] [GitHub Education]
### Hi everyone,

I am writing to bring to your attention an issue I have been facing regarding the activation of my student benefits.

I have successfully completed the student verification process; however, in recent days, I noticed that my student benefits have been deactivated. Consequently, I would like to request a renewal of my application for these benefits.

However, I received a notification from GitHub indicating that I do not need to reverify my student status.
![image](https://github.com/community/community/assets/40395487/5e9ade14-04a1-49ad-9366-6877e821c319)

I kindly request your assistance in resolving this matter as soon as possible. Your prompt attention to this issue would be greatly appreciated.

Thank you for your time and support.

Best regards

**Comment by hellohawaii** [comment]
Which student benefits of you have been deactivated? I think I met a similar problem, I can not use Copilot today, as I stated in [this discussion](https://github.com/orgs/community/discussions/65433#discussion-5574250).

**Reply by Bend1031** [reply]
Yeah, the same problem.
